_maps/RandomRuins/SpaceRuins/crashedship.dmm

```diff
@@ -383,6 +383,7 @@
 /obj/machinery/porta_turret{
 	dir = 8;
 	installation = /obj/item/gun/energy/lasercannon;
+	
 	},
```

Comment:
```suggestion
/obj/machinery/porta_turret{
	dir = 8;
	installation = /obj/item/gun/energy/lasercannon;
	},
```

what da hell newline is this

Reply:
I think the UpdatePaths script added it, I didn't make any manual map changes. Weird.

Reply:
I think it's because whoever made a more recent PR added that semicolon on Line 385, which probably told MapMerge to add in another newline... you aren't supposed to have trailing semicolons in the DMM format. Weird. Just remove that trailing semicolon and new line in this PR for the time being and hopefully it doesn't cause problems to any other updatepaths PRs coming out

Reply:
Delete the semicolon on line 385 as well so we don't keep kicking this issue down the road.